### ISSUE TITLE
Deprecations in TYPO3 12.4 LTS

### ISSUE THREAD

**Issue by dennismetz** [BrainAppeal/campus_events_connector#8]
Hi all,
We are currently updating to the 12 LTS and I am currently getting quite a few deprecations per page view.

Could you provide an update for all extensions that fix this?

Thanks in advance!

**Comment by bahammes** [comment]
Hello @dennismetz ,

we have tested the current releases of all extensions with TYPO3 12.4.14 and PHP 8.3 and couldn't find any deprecation warnings.

Since only EXT:campus_events_frontend has any frontend plugins, the deprecation warnings should originate from there, but we cannot find any problems.

Which PHP version are you using?
Did you update the Campus Events extensions to v4.x?
Can you give us examples of the deprecation warning you are seeing?

Thanks!

**Comment by dennismetz** [comment]
Hi, sorry for the late response. Here is all the info:

PHP8.1
TYPO3 12.4.15 LTS
Latest versions of Campus Events extensions.

The log is filled at least with every call in the backend.
If you need further information, please let me know.

```
The property 'internal_type' of the TCA field 'target_pid' of table 'tx_campuseventsconnector_domain_model_convertconfiguration' is obsolete and has been removed. You can remove it from your TCA as it is not evaluated anymore.
The TCA field 'template_path' of table 'tx_campuseventsconnector_domain_model_convertconfiguration' has been migrated to the TCA type 'folder'. Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_category' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_category' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_category' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_contactperson' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_contactperson' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_contactperson' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_convertconfiguration' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_convertconfiguration' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_event' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_event' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'start_tstamp' of table 'tx_campuseventsconnector_domain_model_event' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'end_tstamp' of table 'tx_campuseventsconnector_domain_model_event' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_event' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_eventattachment' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_eventattachment' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_eventattachment' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_eventimage' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_eventimage' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_eventimage' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_eventsession' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_eventsession' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'start_tstamp' of table 'tx_campuseventsconnector_domain_model_eventsession' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'end_tstamp' of table 'tx_campuseventsconnector_domain_model_eventsession' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_eventsession' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_eventticketpricevariant' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_eventticketpricevariant' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'bookable_from' of table 'tx_campuseventsconnector_domain_model_eventticketpricevariant' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'bookable_till' of table 'tx_campuseventsconnector_domain_model_eventticketpricevariant' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_eventticketpricevariant' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_filtercategory' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_filtercategory' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_filtercategory' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_location' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_location' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_location' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_organizer' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_organizer' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_organizer' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_pricecategory' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_pricecategory' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_pricecategory' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_referent' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_referent' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_referent' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_speaker' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_speaker' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_speaker' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_sponsor' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_sponsor' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_sponsor' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_targetgroup' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_targetgroup' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_targetgroup' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_timerange' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_timerange' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'start_tstamp' of table 'tx_campuseventsconnector_domain_model_timerange' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'end_tstamp' of table 'tx_campuseventsconnector_domain_model_timerange' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_timerange' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'starttime' of table 'tx_campuseventsconnector_domain_model_viewlist' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'endtime' of table 'tx_campuseventsconnector_domain_model_viewlist' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_imported_at' of table 'tx_campuseventsconnector_domain_model_viewlist' defines renderType="inputDateTime". The field has therefore been migrated to the TCA type 'datetime'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_category'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_contactperson'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'status' in table 'tx_campuseventsconnector_domain_model_event'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'min_participants' in table 'tx_campuseventsconnector_domain_model_event'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'max_participants' in table 'tx_campuseventsconnector_domain_model_event'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'participants' in table 'tx_campuseventsconnector_domain_model_event'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'order_type' in table 'tx_campuseventsconnector_domain_model_event'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_event'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_eventattachment'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_eventimage'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_eventsession'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_eventticketpricevariant'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_filtercategory'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_location'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_organizer'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_pricecategory'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_referent'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_speaker'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_sponsor'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_targetgroup'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_timerange'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA field 'ce_import_id' in table 'tx_campuseventsconnector_domain_model_viewlist'" defines eval="int". The field has therefore been migrated to the TCA type 'number'. This includes corresponding migration of the "eval" list, as well as obsolete field configurations, such as "max". Please adjust your TCA accordingly.
The TCA property ['ctrl']['cruser_id'] of table 'tx_campuseventsconnector_domain_model_category'  is not evaluated anymore and has therefore been removed. Please adjust your TCA accordingly.
The TCA property ['ctrl']['cruser_id'] of table 'tx_campuseventsconnector_domain_model_contactperson'  is not evaluated anymore and has therefore been removed. Please adjust your TCA accordingly.
```

**Comment by bahammes** [comment]
Hi @dennismetz ,

thank you for the log details. I misunderstood what deprecations you were referring to.

The TCA migrations were displayed, because the extension works with TYPO3 11 and 12 and we used the same TCA for both versions.

We have added TCA overrides for TYPO3 12 in the latest releases of campus_events_connector  and campus_events_convert2news. There are no more TCA migrations reported in the install tool.
